### ISSUE TITLE
v9.0.2 and use_go_fileserver error download zip file

### ISSUE THREAD

**Issue by Schroeffu** [haiwen/seafile#2537]
I just upgraded v8.0.8 to v9.0.2 including the upgrade_8.0_9.0.sh script, seems to work well except the ZIP Download. ZIP Download is not possible.

Environment:
Ubuntu 20.04 LTS Server
Seafile 9.0.2 CE via seafile download page, not docker 
[fileserver]
use_go_fileserver = true

Exception:
```
2022-01-21 08:55:38,247 [ERROR] django.request:224 log_response Internal Server Error: /api/v2.1/query-zip-progress/
Traceback (most recent call last):
  File "/var/www/clients/client1/web1/private/seafile/seafile-server-9.0.2/seahub/thirdpart/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/var/www/clients/client1/web1/private/seafile/seafile-server-9.0.2/seahub/thirdpart/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/www/clients/client1/web1/private/seafile/seafile-server-9.0.2/seahub/thirdpart/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/var/www/clients/client1/web1/private/seafile/seafile-server-9.0.2/seahub/thirdpart/django/views/generic/base.py", line 70, in view
    return self.dispatch(request, *args, **kwargs)
  File "/var/www/clients/client1/web1/private/seafile/seafile-server-9.0.2/seahub/thirdpart/rest_framework/views.py", line 505, in dispatch
    response = self.handle_exception(exc)
  File "/var/www/clients/client1/web1/private/seafile/seafile-server-9.0.2/seahub/thirdpart/rest_framework/views.py", line 465, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/var/www/clients/client1/web1/private/seafile/seafile-server-9.0.2/seahub/thirdpart/rest_framework/views.py", line 476, in raise_uncaught_exception
    raise exc
  File "/var/www/clients/client1/web1/private/seafile/seafile-server-9.0.2/seahub/thirdpart/rest_framework/views.py", line 502, in dispatch
    response = handler(request, *args, **kwargs)
  File "/var/www/clients/client1/web1/private/seafile/seafile-server-9.0.2/seahub/seahub/api2/endpoints/query_zip_progress.py", line 38, in get
    return Response(json.loads(progress))
  File "/usr/lib/python3.8/json/__init__.py", line 341, in loads
    raise TypeError(f'the JSON object must be str, bytes or bytearray, '
TypeError: the JSON object must be str, bytes or bytearray, not NoneType
```

I found the discussion about a similar or same issue here https://forum.seafile.com/t/seafile-community-edition-9-0-2-is-ready/15569/4 but none of the workarounds extending `export PYTHONPATH` in seahub.sh works for me so far. i may just do something wrong.

Many thanks for checking this issue.

**Comment by olegfusion** [comment]
I confirm this issue

**Comment by killing** [comment]
This is fixed in 9.0.3 version.
